crypt.go

```diff
@@ -304,7 +304,7 @@ func activateWithRecoveryKey(volumeName, sourceDevicePath string, keyReader io.R
 			continue
 		}
 
-		if _, err := unix.AddKey("user", fmt.Sprintf("%s:%s:reason=%d", filepath.Base(os.Args[0]), volumeName, reason), key[:], userKeyring); err != nil {
+		if _, err := unix.AddKey("user", fmt.Sprintf("secboot-recover:%s?reason=%d", sourceDevicePath, reason), key[:], userKeyring); err != nil {
```

Comment:
I'm happy this moved away from os.Args[0], but I still think we want a way to let the caller control these prefixes. I still think we need to have a callback mechanism for this.